vllm/v1/engine/processor.py

```diff
@@ -394,7 +394,26 @@ def process_inputs(
         trace_headers: Mapping[str, str] | None = None,
         priority: int = 0,
         data_parallel_rank: int | None = None,
+        close_streaming_session: bool | None = None,
     ) -> EngineCoreRequest:
+        # For close_streaming_session requests, skip input preprocessing
+        # and create a minimal request to signal session closure
+        if close_streaming_session:
+            return EngineCoreRequest(
+                request_id=request_id,
+                prompt_token_ids=[],
+                mm_features=None,
+                sampling_params=params if isinstance(params, SamplingParams) else None,
+                pooling_params=params if isinstance(params, PoolingParams) else None,
+                eos_token_id=None,
+                arrival_time=arrival_time if arrival_time else time.time(),
+                lora_request=lora_request,
+                cache_salt=None,
+                priority=priority,
+                data_parallel_rank=data_parallel_rank,
+                close_streaming_session=True,
+            )
+
```

Comment:
For non-streaming requests, do we default `close_streaming_session` to `None` or `True`?  I personally prefer `True` and not distinguishing between streaming and non-streaming requests. We can view a non-streaming request as a request that a user has finished their input in the first round.